code/__HELPERS/unsorted.dm

```diff
@@ -1214,3 +1214,24 @@ proc/shift_proximity(atom/checker, atom/A, range, atom/B, newrange)
 		var/turf/F = E
 		F.proximity_checkers |= checker
 	return 1
+
+/proc/flick_overlay_better(image/I, atom/A, duration) //todo :^) change flick_overlay() to this
+	set waitfor = 0
+	if(!A || !I)
+		return
+	A.overlays |= I
+	sleep(duration)
```

Comment:
Call it uh, flick_overlay_static() or flick_overlay_no_animate(), something descriptive about the quirks of this version.